### PR TITLE
[content-bundle] change doc to best practices

### PR DESCRIPTION
**Pull request by dbu**

### CODE REVIEW

File: bundles/content/exposing_content_via_rest.rst [hunk 1]
```diff
@@ -72,6 +73,7 @@ Here is an example configuration for the FOSRestBundle.
     .. code-block:: xml
 
         <?xml version="1.0" encoding="UTF-8" ?>
+        <!-- app/config/config.xml -->
```
Comment (wouterj) [inline bundles/content/exposing_content_via_rest.rst]: should be before the doctype thing

Comment (dbu) [inline bundles/content/exposing_content_via_rest.rst]: oh, ok. if that is the convention, i can change. but that would mean its not valid xml anymore. the doctype must be the very first thing in an xml file afaik.

Comment (wouterj) [inline bundles/content/exposing_content_via_rest.rst]: Yeah, that's correct. However, one is not supposed to copy the file comment (e.g. no PHP file is valid as well, as there is no `<?php`).

**Comment by dbu** [comment]
fixed and rebased in #810

**Pull request closed without merging**